json-rpc/types/src/views.rs

```diff
@@ -371,12 +371,8 @@ pub enum VMStatusView {
         function_index: u16,
         code_offset: u16,
     },
-    #[serde(rename = "verification_error")]
-    VerificationError,
-    #[serde(rename = "deserialization_error")]
-    DeserializationError,
-    #[serde(rename = "publishing_failure")]
-    PublishingFailure,
+    #[serde(rename = "uncategorized_program_error")]
+    UncategorizedProgramError,
```

Comment:
please update doc https://github.com/libra/libra/blob/master/json-rpc/docs/type_transaction.md#verification_error

and add change to https://github.com/libra/libra/blob/master/json-rpc/API-CHANGELOG.md